### ISSUE TITLE
Next button on Subjects of Interest screen

### ISSUE THREAD

**Issue by shaunan** [StatCan/StatcanPublicationUAT#597]
During onboarding, on the “What interests you” screen, the “Next” button is disabled/greyed out initially, however, after selecting some subjects and deselecting them, the “Next” button remains in the active state even if no subjects are selected. Selecting the “Next” button does not perform any action, and the user cannot proceed.

If the user cannot continue without having made a selection, or has removed previously made selections, then either disable the “Next” button, and/or implement a status message to inform the user that they must either make a selection to continue or select the “Skip” link to ignore this page. 

![GitHub10](https://github.com/StatCan/StatcanPublicationUAT/assets/115822055/a37e4787-5462-4434-9e14-ed26b23726e6)
